### DISCUSSION TITLE
유니티의 animation evenet로 함수를 호출하고 싶은데...

### DISCUSSION THREAD

**Discussion by cinephile231** [naverz/zepeto-studio-kor#897] [5. Q&A]
호출이 안 되네요. 방법이 있을까요?

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 어떤 코드를 사용하셨고 어떤 부분에서 어려움을 겪었는지 보다 상세하게 작성해주시면
답변에 도움을 드릴 수 있을 것 같습니다.

**Comment by HenryAtBraveTurtles** [comment]
글 작성자는 아니지만 비슷한 어려움을 겪어 댓글 답니다~
https://docs.unity3d.com/Manual/script-AnimationWindowEvent.html
위 문서 참고하시면 애니메이션 이벤트로 C# 함수를 호출할 수 있는 기능이 있는데 이 기능이 제페토 스크립트 함수와는 연결이 안되서 어려운 점이 있었습니다. 
아마도 이것와 비슷한 질문하시는 것 같네요 ㅎㅎ
